docker-compose.yml

```diff
@@ -26,6 +23,31 @@ services:
       rabbitmq:
         condition: service_healthy
 
+  php-cli:
+      container_name: php.uitdatabank
+      extends:
+          service: php
+      profiles:
```

Comment:
Is this correct, should this not always run? I think this breaks the existing setup when not using the xdebug profile